src/attrib.d

```diff
@@ -394,6 +394,7 @@ extern (C++) final class DeprecatedDeclaration : StorageClassDeclaration
 {
 public:
     Expression msg;
+    const(char)* message;
```

Comment:
`msg` vs `message` is a little confusing. I want to suggest either:
1. rename original `msg` field to `emsg`
2. rename new `message` field to `msgstr`
or something better.